### PR TITLE
Adaugă schiță laborator Îmbunătățirea lucrului în linia de comandă

### PR DESCRIPTION
**Pull request by edi33416**

### CODE REVIEW

**Comment by Sergiu121** [comment]
Publised at http://sergiu121.github.io/refs/pull/5/merge

**Comment by Sergiu121** [comment]
Publised at http://sergiu121.github.io/refs/pull/5/merge

File: chapters/improve-cli/improve-cmds.rst [hunk 1]
```diff
@@ -0,0 +1,183 @@
+#######################################
+O folosire mai bună a comenzilor uzuale
+#######################################
+
+************************************
+1) Inspectarea sistemului de fișiere
```
Comment (CostinCarabas) [inline chapters/improve-cli/improve-cmds.rst:6]: Dacă este un singur punct, nu are rost numerotarea.

**Comment by Sergiu121** [comment]
Publised at http://sergiu121.github.io/refs/pull/5/merge

**Comment by Sergiu121** [comment]
Publised at http://sergiu121.github.io/refs/pull/5/merge

**Comment by Sergiu121** [comment]
Publised at http://sergiu121.github.io/refs/pull/5/merge

**Comment by Sergiu121** [comment]
Publised at http://sergiu121.github.io/refs/pull/5/merge

**Comment by Sergiu121** [comment]
Publised at http://sergiu121.github.io/refs/pull/5/merge

**Comment by Sergiu121** [comment]
Publised at http://sergiu121.github.io/refs/pull/5/merge

**Comment by Sergiu121** [comment]
Publised at http://sergiu121.github.io/refs/pull/5/merge

**Pull request merged**